ethcore/sync/src/chain/mod.rs

```diff
@@ -676,6 +691,8 @@ pub struct ChainSync {
 	sync_start_time: Option<Instant>,
 	/// Transactions propagation statistics
 	transactions_stats: TransactionsStats,
```

Comment:
Do we need changes here too to account for `eth/65` traffic? Would be good to have good data on protocol usage.